sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -188,7 +193,7 @@ public void run() {
         }
 
         try {
-          ReadableSpan lastElement = queue.poll(100, TimeUnit.MILLISECONDS);
+          ReadableSpan lastElement = queue.poll(queuePollWaitTimeNanos, TimeUnit.NANOSECONDS);
```

Comment:
Can you document the motivation? We don't want to add configuration for everything unless there is a good use-case to configure that.